CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -100,6 +100,11 @@ class HomeViewController: UIViewController {
         super.viewWillAppear(animated)
         navigationController?.setNavigationBarHidden(true, animated: true)
     }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        navigationController?.setNavigationBarHidden(false, animated: false)
```

Comment:
ooo yea if you're setting this beforehand, i believe you don't have to explicitly set `.isHidden` to false in each of the individual view controllers!